src/Core/Logger.php

```diff
@@ -64,7 +64,6 @@ class Logger extends BaseObject
 		self::TRACE => 'Trace',
 		self::DEBUG => 'Debug',
 		self::DATA => 'Data',
-		self::ALL => 'All',
```

Comment:
How is this entry a duplicate?

Reply:
Look at the definition of the constants, both `self::DATA` and `self::ALL` equal `debug`, which makes them unsuitable to be keys in the same array.